{
    "affected": [
        {
            "package": {
                "ecosystem": "photon:1.0",
                "name": "python-py",
                "purl": "pkg:rpm/vmware/python-py?distro=photon-1"
            }
        }
    ],
    "id": "PHSA-2021-0419",
    "modified": "2023-03-16T05:25:30Z",
    "published": "2021-07-27T00:00:00Z",
    "references": [
        {
            "type": "ADVISORY",
            "url": "https://github.com/vmware/photon/wiki/Security-Update-1.0-419"
        }
    ],
    "related": [
        "CVE-2020-29651"
    ]
}